### PR TITLE
Diletant attempt to make the page less ugly

### PR DESCRIPTION
**Pull request by suhr**
Now eye bleeding is a bit less intensive.

### CODE REVIEW

File: css/screen.css [hunk 1]
```diff
@@ -1,13 +1,15 @@
...
     -webkit-text-size-adjust: none;
 }
 
 body {
-    color: #000000;
+    color: #222;
     background-color: rgb(242, 242, 242);
     text-align: center;
     font-family: Helvetica, Arial, sans-serif;
+    font-size: 16px;
```
Comment (sebix) [inline css/screen.css]: I would prefer `em` instead of `px`.

Comment (suhr) [inline css/screen.css]: I will remove this line since browsers have 16px as default size anyway.

Comment (sebix) [inline css/screen.css]: Ah, that's the definition `1em = 16px` (or one part of it).

**Comment by sebix** [comment]
I'll have a more detailed look today evening, thanks for your suggestions! :)

**Comment by sebix** [comment]
Thanks, merged!

If you use Winpython you may consider testing beta releases and join the mailinglist if you haven't done so already. There are only 2 testers AFAIK.

**Pull request merged**